File: Python/examples/PaymentAPI/getAccountInfo.py
import os
import os.path
import sys
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir + os.path.sep + os.path.pardir)))
from dotenv import load_dotenv
import json

load_dotenv(os.path.pardir + os.sep + ".env")
eid = os.getenv("EID")
secret = os.getenv("SECRET")

from PaymentAPI import PaymentAPI

# Create a PaymentAPI object
api = PaymentAPI(eid, secret)
account = api.call(function="getAccountinfo")
print(json.dumps(account, indent=4))

File: Deno/examples/PaymentAPI/getAccountInfo.ts
import {QvicklyPaymentAPI, env} from "../../PaymentAPI.ts";

const paymentAPI = new QvicklyPaymentAPI(env["EID"], env["SECRET"]);
const account = await paymentAPI.call("getAccountInfo", {
    "comingFromPF2": "1"
});
console.log(account);
